### PR TITLE
Chore change zero signature to factory method sign privkey

### LINKED ISSUE #1058: Change `ZERO_SIGNATURE` to factory method `sign__privkey()`
Now in tests we init messages signature using `fake_sig()`:

```python
report_computed_task = factories.tasks.ReportComputedTaskFactory(
        sig=ZERO_SIGNATURE,
        task_to_compute=task_to_compute,
    )
```

Factories from Golem Messages contains method to signing messages during it initialization. All places where `ZERO_SIGNATURE` is used should be changed to:
```python
report_computed_task = factories.tasks.ReportComputedTaskFactory(
        sign__privkey=PROVIDER_PRIVATE_KEY,
        task_to_compute=task_to_compute,
    )

### PR DESCRIPTION
**Pull request by kbeker**
Resolves #1058

### CODE REVIEW

File: concent_api/core/tests/test_core_views.py [hunk 1]
```diff
@@ -558,34 +556,33 @@ class CoreViewReceiveTest(ConcentIntegrationTestCase):
     def setUp(self):
         with freeze_time("2017-11-17 10:00:00"):
             super().setUp()
-            self.compute_task_def = ComputeTaskDefFactory()
+            self.compute_task_def = tasks.ComputeTaskDefFactory()
             self.compute_task_def['deadline'] = get_current_utc_timestamp() + (60 * 37)
-            self.want_to_compute_task = WantToComputeTaskFactory(
+            self.want_to_compute_task = tasks.WantToComputeTaskFactory(
```
Comment (dybi) [inline concent_api/core/tests/test_core_views.py]: are all those params below necessary? If not, we could reduce the boilerpalte ;)

File: concent_api/core/tests/test_core_views.py [hunk 2]
```diff
@@ -832,36 +829,36 @@ class CoreViewReceiveOutOfBandTest(ConcentIntegrationTestCase):
 
     def setUp(self):
         super().setUp()
-        self.compute_task_def = ComputeTaskDefFactory()
+        self.compute_task_def = tasks.ComputeTaskDefFactory()
         self.compute_task_def['deadline'] = get_current_utc_timestamp() - 60
-        self.want_to_compute_task = WantToComputeTaskFactory(
+        self.want_to_compute_task = tasks.WantToComputeTaskFactory(
```
Comment (dybi) [inline concent_api/core/tests/test_core_views.py]: same here

File: concent_api/core/tests/test_unit_custom_manager_queryset.py [hunk 3]
```diff
@@ -4,10 +4,10 @@
 import pytest
 
 from golem_messages import constants
-from golem_messages.factories.tasks import ReportComputedTaskFactory
+from golem_messages.factories.tasks import ReportComputedTaskFactory, TaskToComputeFactory
```
Comment (Jakub89) [inline concent_api/core/tests/test_unit_custom_manager_queryset.py]: uuuuu, You know what's up :D

File: concent_api/core/tests/test_core_views.py (hunk 1 above)
Comment (kbeker) [reply concent_api/core/tests/test_core_views.py]: That's true, deleted :)

File: concent_api/core/tests/test_core_views.py (hunk 2 above)
Comment (kbeker) [reply concent_api/core/tests/test_core_views.py]: That's true, deleted :)

File: concent_api/core/tests/test_unit_custom_manager_queryset.py (hunk 3 above)
Comment (kbeker) [reply concent_api/core/tests/test_unit_custom_manager_queryset.py]: UPS, fixed :P

**Comment by kbeker** [comment]
download
 i siup merge

**Comment by PaweuB** [comment]
`Siup` is the new merge!

**Pull request merged**